src/styles/blocks/card.scss

```
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+
+  display: flex;
+  flex-direction: column;
+
+  border-radius: 5px;
+  border: 1px solid $card-border;
+  background: $white-color;
+
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  font-family: Roboto, sans-serif;
+
+  &__image {
+    margin: 0 auto 36px;
+  
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px; /* 150% */
+    margin: 4px 0;
+  }
+
+  &__code {
+    color: $secondary;
+    font-size: 10px;
+    line-height: 14px; /* 140% */
+    margin: 0 0 16px;
```

you can just put margin-bottom here
```suggestion
    margin-bottom: 16px;
```